.reuse/dep5

```diff
@@ -0,0 +1,7 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Contact: Digital Bazaar, Inc. <support@digitalbazaar.com>
+
+Files: *.md .gitignore .npmrc package.json test/.npmrc test/mocha/.eslintrc
+    test/package.json .editorconfig .github/* test/static/*.html
+Copyright: 2012-2024 Digital Bazaar, Inc.
```

Comment:
Git history says 2013.
```suggestion
Copyright: 2013-2024 Digital Bazaar, Inc.
```

Reply:
Several files state their copyright is 2012, so I went by that rather than the git log--as copyright reaches beyond such ephemeral limitations. 😉 

That said, I've no way of knowing which is more correct... Presumably the original author knows? @dlongley?

Most of these repos have an initial commit date of December 31, 2013 according to:
```sh
git log --reverse --all | head -n 3
```

Reply:
Not gonna change copyright years to match git logs as copyright exists beyond git, so marking this one as resolved.